integration-tests/jpa-mariadb/pom.xml

```diff
@@ -29,6 +29,11 @@
     <artifactId>quarkus-integration-test-jpa-mariadb</artifactId>
     <name>Quarkus - Integration Tests - JPA - MariaDB</name>
     <description>Module that contains JPA related tests running with the MariaDB database</description>
+
+    <properties>
+        <mariadb.url>jdbc:mariadb://localhost:3308/hibernate_orm_test</mariadb.url>
```

Comment:
I think you should use the default port in the default case. The port is specifically adjusted when you enable our Docker config.

Feel free to ignore if it doesn't make sense to you :).

Reply:
Ok, since it's just an example that might help. Changing.